all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -57,7 +57,7 @@ objects:
       spec:
           containers:
             - name: ${JAEGER_NAME}
-              image: jaegertracing/all-in-one:f209812d
+              image: jaegertracing/all-in-one:9f3984cd
```

Comment:
If you were using `latest`, you wouldn't need this update ;-) This isn't a real version, right? So, before approving, I'd ask to figure out the versioning strategy based on the broader feedback you received on this subject. We can use the same for the other PR.